javascript/ql/src/semmle/javascript/security/dataflow/RemotePropertyInjection.qll

```diff
@@ -70,15 +70,25 @@ module RemotePropertyInjection {
       result = " a property name to write to."
     }     
   }
+
+  /**
+   * Holds if the method name injection on the given base object is handled by another query.
+   */
+  private predicate isCoveredByMethodNameInjection(DataFlow::SourceNode node) {
+    node = DataFlow::globalObjectRef()
+    or
+    node.analyze().getAValue() instanceof AbstractCallable
```

Comment:
Do we also want to whitelist nodes that have an invocation here? (And could we share this predicate with the other query?)